application/models.py

```diff
@@ -13,7 +12,7 @@
 
 
 class HighSchoolApplication(models.Model):
-    application_number = models.CharField(max_length=20, unique=True)
+    application_number = models.CharField(max_length=20)
```

Comment:
Curious about this change a bit. So we'll be having all applications from a student with the same number?

Reply:
No, this is to make sure that the student can apply to the same school and program, say after 6 months.